invenio_app_rdm/theme/templates/semantic-ui/invenio_app_rdm/help/search.en.html

```diff
@@ -184,12 +184,12 @@ <h3>Missing values</h3>
   <p>
     It is possible to search for records that either are missing a value or have
     a value in a specific field using the <code>_exists_</code> and
-    <code> not _exists_</code> field names.
+    <code>NOT _exists_</code> field names.
   </p>
   <p>
     <strong>Example:</strong>
-    <a href="/search?page=1&amp;size=20&amp;q=_missing_:metadata.additional_titles"
-      ><code>not _exists_:metadata.additional_titles</code></a
+    <a href="/search?page=1&amp;size=20&amp;q=NOT%20_exists_:metadata.additional_titles"
+      ><code>NOT _exists_:metadata.additional_titles</code></a
     >
```

Comment:
very minor - can we clean up the HTML there?

```suggestion
    <a href="/search?page=1&amp;size=20&amp;q=NOT%20_exists_:metadata.additional_titles">
      <code>NOT _exists_:metadata.additional_titles</code>
    </a>
```